package.json

```diff
@@ -28,10 +28,10 @@
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
+    "@filipeop/notifications-panel": "^0.0.2",
```

Comment:
Please increase the "version": "0.0.7", to 0.0.8 so we can release and publish for Dynamo